Main.java

```diff
@@ -9,7 +10,21 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    
+    //create a Scanner for user input
+   Scanner input = new Scanner(System.in);
+    
+
+    // ask user for temp
+    System.out.println("Please enter a temperature you would like to convert in degrees Celsius.");
+
+    // variable Celsius is the temp
+    int C = input.nextInt();
+
+  // Declare and calculate the temp
+  int F=(C*9)/5 +32;
```

Comment:
Good work with the commenting in your code. Remember that variable names should always begin with a lowercase letter and be in the camelCase format. While C and F are the units  attached to the value, it really doesn't describe what it is storing. Something like degreesC or tempInC would be a much better name